ci/docker/runtime_functions.sh

```diff
@@ -1580,9 +1551,404 @@ deploy_docs() {
 
     export CC="ccache gcc"
     export CXX="ccache g++"
-    make docs SPHINXOPTS=-W USE_MKLDNN=0
 
+    build_python_docs
+
+    popd
+}
+
+
+build_docs_setup() {
+    build_folder="docs/_build"
```

Comment:
will you be adding this  : https://github.com/ThomasDelteil/incubator-mxnet/pull/6#discussion_r321536377

Reply:
actually this is called from other functions, does not require the set -ex